### ISSUE TITLE
startxfce4  fails to open desktop

### ISSUE THREAD

**Issue by SJW53** [dnschneid/crouton#4362]
```
Please paste the output of the following command here: sudo edit-chroot -all
```
name: xenial
encrypted: no
Entering /mnt/stateful_partition/crouton/chroots/xenial...
WARNING: Unable to register core dump handler.
crouton: version 1-20181001133934~master:38012fdf
release: xenial
architecture: amd64
xmethod: xiwi
targets: extension,keyboard,audio,touch,xfce,xiwi
host: version 13310.93.0 (Official Build) stable-channel cave 
kernel: Linux localhost 3.18.0-20518-gb953b77383b8 #1 SMP PREEMPT Thu Sep 24 20:26:34 PDT 2020 x86_64 x86_64 x86_64 GNU/Linux
freon: yes
Unmounting /mnt/stateful_partition/crouton/chroots/xenial...
chronos@localhost / $

#### Please describe your issue:

I installed crouton, xenial and xfce on my Chromebook some time ago, and all has been working well until today when I went to exit the desktop and accidentally hit 'Reset' rather than 'Exit'. I cannot now re-enter the desktop.

#### If known, describe the steps to reproduce the issue:

When I enter:
   `sudo startxfce4`

This text flashes up on the screen:

    `Entering /mnt/stateful_partition/crouton/chroots/xenial...
    WARNING: Unable to register core-dump handler... `

Then the screen refreshes and displays:

    `D9R16C10R16C11R16C12R16C13R16C14R16C15R16C24R1C0^[[?1;2cUnmounting /mnt/stateful_partition/crouton/chroots/xenial...`
    `Sending SIGTERM to processes under /mnt/stateful_partition/crouton/chroots/xenial...`
    `chronos@localhost / $ 1;2`

I would appreciate any suggestions on how to overcome this problem.
Thanks, Stephen

**Comment by DennisLfromGA** [comment]
@SJW53,

Your crouton version is ancient:
> crouton: version 1-20181001133934~master:38012fdf

Please download a fresh copy from [here](https://goo.gl/fd3zc)
Then follow these instructions to install it:
> Download it, pop open a shell (Ctrl+Alt+T, type shell and hit enter), make the installer executable with sudo install -Dt /usr/local/bin -m 755 ~/Downloads/crouton, then launch it with sudo crouton to see the help text. 

Next update you chroot by entering this:
- `sudo crouton -n xenial -u`

*Hope this helps,*
-**DennisLfromGA**

**Comment by SJW53** [comment]
Hi Dennis,

I updated using your instructions, and it completed fully. Howver,  I get exactly the same error as before when I run:
`sudo startxfce4`

Maybe I should fully delete 'xfce' and re-install it? Can you give me the commands for this? Any other suggestions? Thanks.

**Comment by MrHallway0** [comment]
To remove a chroot run `sudo edit-chroot -d <CHROOT NAME>`.
If you didn't change the name (`-n`) during install then it should just be `sudo edit-chroot -d xenial`.

**Comment by SJW53** [comment]
I don't want to delete the chroot as it has at lot of files and setup that I made for an application.

The problem relates to xfce. Can I delete xfce and leave the chroot otherwise intact, and then reload xfce?

**Comment by MrHallway0** [comment]
`sudo enter-chroot` and then you can run the `apt` commands to remove and reinstall the desktop.
I believe the package is `xfce4`...

**Comment by SJW53** [comment]
Thanks. 
Once I realised that I could access the files in the chroot from the command line in the crosh shell, I decided I would create a new chroot and then manually copy the files required by the old software and database that I am using from the old chroot to the new chroot.
I started this a few hours ago, and this has worked OK.
I did find some very old stuff on the xfce website that referred to similar issues, and no resolution was found then, It seems that you must always use the 'log out' option to exit the xfce desktop. The other options offered such as 'restart' and 'shutdown' can result in blocking all future access like I unfortunately discovered. 
Thanks for your help. It is much apprciated,
